mono/mini/mini-runtime.c

```diff
@@ -4738,7 +4738,7 @@ mono_runtime_print_stats (void)
 		g_print ("JIT info table removes: %" G_GINT32_FORMAT "\n", mono_stats.jit_info_table_remove_count);
 		g_print ("JIT info table lookups: %" G_GINT32_FORMAT "\n", mono_stats.jit_info_table_lookup_count);
 
-		mono_counters_dump (MONO_COUNTER_SECTION_MASK | MONO_COUNTER_MONOTONIC, stdout);
+		mono_counters_dump (MONO_COUNTER_SECTION_MASK | MONO_COUNTER_MONOTONIC, NULL);
```

Comment:
This will segfault until @CoffeeFlux 's changes are back ported.

Reply:
Changed are already backported, but try rebasing to make sure.